templates/messages/messages.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  commandDescription: "print a greeting and your org IDs",
+  nameFlagDescription: "name to print",
+  forceFlagDescription: "example boolean flag",
+  errorNoOrgResults: "No results found for the org '%s'.",
+  example1: `$ sfdx hello:org --targetusername myOrg@example.com --targetdevhubusername devhub@org.com
```

Comment:
I have hardcoded <%- topicName %>:<%- commandName %> to hello:org. I have tried to use string interpolation, but it throws an error after creating a plugin as topicName and commandName will not be present in the created org.ts file.
LMK what do you think...

Reply:
here's an example of what messages usually look like in current-generation plugins : https://github.com/salesforcecli/plugin-user/tree/main/messages

let's encourage people to do it that way instead of like this (older, toolbelt style https://github.com/salesforcecli/toolbelt/blob/main/messages/mdapi_cancel.js)

Reply:
and here's an example of how those multiline [array] examples are used https://github.com/salesforcecli/plugin-user/blob/8366e79135c4d15bbb6f205e8dfa631d2b83d713/src/commands/force/user/create.ts#L59

This is a definitely a case of us not reading the WI details very carefully.

Can you just ignore everything in the WI and "make the example look like plugin-user" where it uses the json file, handles multiline examples an array of string via `.split(os.EOL)` or whatever.

Hopefully that makes sense.